Watchman.Discord/Areas/Initialization/Services/ResponsesInitService.cs

```diff
@@ -23,28 +23,36 @@ public ResponsesInitService(ICommandBus commandBus, ResponsesGetterService respo
         public async Task InitNewResponsesFromResources()
         {
             var defaultResponses = this._responsesGetterService.GetResponsesFromResources();
-            var responsesInBase = this._responsesGetterService.GetResponsesFromBase().Select(response =>
+            
+            var responsesToUpdate = this._responsesGetterService.GetResponsesFromBase().Select(baseResponse =>
             {
                 // Update every response in the DB with availableVariables from defaultResponses
-                return new Response(response.OnEvent, response.Message, response.ServerId, 
-                    defaultResponses.First(responseNew => responseNew.OnEvent == response.OnEvent).AvailableVariables);
-            });
+                var matchingDefaultResponse = defaultResponses
+                    .FirstOrDefault(defaultResponse => defaultResponse.OnEvent == baseResponse.OnEvent);
+                if (matchingDefaultResponse != null)
+                {
+                    baseResponse.UpdateAvailableVariables(matchingDefaultResponse.AvailableVariables);
+                    baseResponse.SetMessage(matchingDefaultResponse.Message);
+                }
+                return baseResponse;
+            }).ToArray();
+            
             var responsesToAdd = defaultResponses
-                .Where(def => responsesInBase.All(@base => @base.OnEvent != def.OnEvent))
-                .ToList();
+                .Where(def => responsesToUpdate.All(@base => @base.OnEvent != def.OnEvent))
+                .ToArray();
```

Comment:
dlaczego nie ToList?
wydajność jest bardzo porównywalna (lekka przewaga dla ToList) ale nie o to chodzi
chodzi mi, że w większości miejsc używamy ToList i jakoś tak dziwnie wygląda ToArray